### DISCUSSION TITLE
Bundle-RequiredExecutionEnvironment in MANIFEST.MF changes from JavaSE-17 to J2SE-1.5 after building

### DISCUSSION THREAD

**Discussion by johannesctrl** [eclipse-emf/org.eclipse.emf#7] [Q&A] [answered]
I currently have the issue that the `Bundle-RequiredExecutionEnvironment` in my MANIFEST.MF changes from `JavaSE-17` to `J2SE-1.5` after the project was built in my Eclipse IDE, although I want it to stay as `JavaSE-17`.

Steps to reproduce:
 - Open Eclipse 2023-06 (Installed JRE = jdk-17.0.4)
 - Add `org.eclipse.emf.ecore.xcore` as Required-Bundle to MANIFEST.MF
 - Add buildCommand `<name>org.eclipse.xtext.ui.shared.xtextBuilder</name>` and nature `<nature>org.eclipse.xtext.ui.shared.xtextNature</nature>` to the .project file
 - Create an Example.xcore file
 - Let Eclipse auto-build the project

After the building the diff of the MANIFEST.MF looks like that:
![image](https://github.com/eclipse-emf/org.eclipse.emf/assets/81531876/52e9ca1a-2ad5-450c-9b70-89bd4a0d7fc6)

Please check out my [Example Repository](https://github.com/johannesctrl/bugreport-xcore-bundle-requiredExecutionEnvironment) that reproduces this behavior. What do you guys think about that - does it work as intended or is it a bug? I would also be thankful if someone could send me a link to setup Xcore for debugging and contribution purposes if it turns out to be a bug.

**Comment by merks** [comment, marked as answer]
You should change the Java compliance property in the xcore model.  This is easiest done via the properties view with the package selected to create the right annotation.

**Reply by ingomohr** [reply]
Thank you @merks ! Works like a charm. :)
